processor/sourcemap/payload.go

```diff
@@ -16,23 +16,25 @@ var (
 	processorEntry   = common.MapStr{"name": processorName, "event": smapDocType}
 )
 
-type payload struct {
+type Payload struct {
 	ServiceName    string
 	ServiceVersion string
 	Sourcemap      string
 	BundleFilepath string
+
+	config config.Config
```

Comment:
might be a bit fiddly to put configuration and user data in the same struct...

Reply:
You are right, going to change that.